sut/drush/sites/sut.site.yml

```diff
@@ -0,0 +1,5 @@
+common:
+  root: /Users/ganderson/local/drupal/drush9/drush/sut
```

Comment:
Is this a mistake? If not please add a comment. My path is different. 

Also add link to wildcard aliases docs.

Reply:
That's a mistake.